Movies_MVVM/Modules/MoviesListTab/View/MovieListViewController.swift

```diff
@@ -17,6 +17,8 @@ final class MovieListViewController: UIViewController {
         static let emptyString = ""
         static let titleApiAlert = "Введите API ключ для загрузки данных"
         static let loadingButtonTitle = "Загрузить"
+        static let moviesCollectionViewID = "Movies"
+        static let topRatedButtonID = "Top"
```

Comment:
```suggestion
        static let topRatedButtonID = "Top"
```
К кнопке можно достучаться по ее имени, не только по ID. Это не замечание, просто рассуждение